site/content/en/docs/Installation/Terraform/aks.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Installing Agones on Azure Kubernetes Service using Terraform"
+linkTitle: "Azure"
+weight: 20
+description: >
+  You can deploy Kubernetes cluster on Azure Kubernetes Service and install Agones using Terraform.
+---
+
+## Installation
+
+Install `az` utility by following [these instructions](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest).
+
+The example of AKS submodule configuration could be found here:
+ {{< ghlink href="examples/terraform-submodules/aks/module.tf" >}}Terraform configuration with Agones submodule{{< /ghlink >}}
+
+Copy `module.tf` file into a separate folder.
+
+Login to Azure CLI:
+```
+az login
+```
+
+Configure your terraform:
+```
+terraform init
+```
+
+Now you can deploy your cluster (use variables from the above `az ad sp create-for-rbac` command output):
+```
+terraform apply -var client_id="<appId>" -var client_secret="<password>"
+```
+
+Once you created all resources on AKS you can get the credentials so that you can use `kubectl` to configure your cluster:
+```
+az aks get-credentials --resource-group agonesRG --name test-cluster
+```
+
+Check that you have access to kubernetes cluster:
+```
+kubectl get nodes
+```
+
+### Uninstall the Agones and delete AKS cluster
```

Comment:
This is super minor, but for consistency I think this should be ## instead of ###. Same for the reference section.

Reply:
This file didn't change in the latest version. Did you miss pushing a commit?

Reply:
Hmmm. The file changed but the code review tool doesn't show that it's new and that I hadn't looked at it.